sway/desktop/output.c

```diff
@@ -114,13 +114,14 @@ static void output_for_each_surface_iterator(struct wlr_surface *surface,
 }
 
 void output_surface_for_each_surface(struct sway_output *output,
-		struct wlr_surface *surface, double ox, double oy,
-		sway_surface_iterator_func_t iterator, void *user_data) {
+		struct wlr_surface *surface, struct sway_view *view,
+		double ox, double oy, sway_surface_iterator_func_t iterator,
+		void *user_data) {
 	struct surface_iterator_data data = {
 		.user_iterator = iterator,
 		.user_data = user_data,
 		.output = output,
-		.view = NULL,
+		.view = view,
 		.ox = ox,
 		.oy = oy,
 		.width = surface->current.width,
```

Comment:
Thought: could we just keep this function as is, but instead have the ox / oy / width / height as view->x, view->y, view->width, view->height? Could clean things up a bit, not sure we'd need the surface dimensions